packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -285,24 +288,29 @@ function ViewGrid< Item >( {
 		{ regularFields: [], badgeFields: [] }
 	);
 	const hasData = !! data?.length;
-	const updatedPreviewSize = useUpdatedPreviewSizeOnViewportChange();
 	const hasBulkActions = useSomeItemHasAPossibleBulkAction( actions, data );
-	const usedPreviewSize = updatedPreviewSize || view.layout?.previewSize;
-	const gridStyle = usedPreviewSize
-		? {
-				gridTemplateColumns: `repeat(${ usedPreviewSize }, minmax(0, 1fr))`,
-		  }
-		: {};
+	const usedPreviewSize = view.layout?.previewSize;
+
+	// Calculate possible media sizes in grid for responsive images.
+	let sizes = '400px';
```

Comment:
I'm now wondering whether it's really worth switching the value of sizes for different grid sizes, because it means that changing the grid size in the UI can download a whole new set of images 😅 

We could just go with a fixed maximum size of 900px here, because with `grid-template-columns` using `auto-fill`, even the biggest grid size on a really big screen won't go over that.

Reply:
Good rationale 👍